package.json

```diff
@@ -32,6 +32,9 @@
     "@lingui/react": "^2.7.0",
     "@reach/router": "^1.1.1",
     "classnames": "^2.2.6",
+    "glhd-tailwindcss-transitions": "^0.3.0",
+    "i18next": "^11.7.0",
```

Comment:
Please remove `i18next` and `i18next-browser-languagedetector` we are not using these anymore